resonate/scheduler.py

```diff
@@ -802,7 +802,7 @@ def _eval(self, node: Node[State]) -> list[Function | Delayed[Function | Retry]
                         return []
 
                     case RFI(conv), Enabled(Suspended(Init(next=None))):
-                        next = Rfnc(conv.id, self.id, conv, clock.time() + conv.timeout)
+                        next = Rfnc(conv.id, self.id, conv, min(clock.time() + conv.timeout, c.timeout))
```

Comment:
We just need to exempt detached from the min check, this information is in the mode parameter of the RFI

Reply:
Added!